src/montecarlo/run_fund_mcs.jl

```diff
@@ -17,13 +17,12 @@ function run_fund_mcs(trials = 10000; ntimesteps = MimiFUND.default_nsteps + 1,
     mcs = getmcs()
 
     # Generate trials
-    if save_trials
-        generate_trials!(mcs, trials; filename = joinpath(output_dir, "trials.csv"))
-    end
+    filename = save_trials ? joinpath("$output_dir/trials.csv") : ""
+    generate_trials!(mcs, trials; filename=filename)
```

Comment:
Not related to this PR, but this strikes me as a strange API: is an empty string treated in some special way when passed to `filename`? If so, I think the sentinel value to not save to a file should be `nothing`, not an empty string.

CC @lrennels

Reply:
this used to be the case, which I agree is weird, but it's possible that this has been changed since the last time I worked with generate_trial code. I'll check

Reply:
looks like this is still the API. I'll open an issue on Mimi

Reply:
https://github.com/mimiframework/Mimi.jl/issues/444

Reply:
I would prefer to fix this first in Mimi.jl, and then only have  PR here that works with the final design in Mimi.jl, to keep the churn here to a minimum.

Reply:
@davidanthoff I can do that later today.  Since we aren't doing too many breaking changes this doesn't generally come up, but I tend to run into these circular testing issues between our dependent Mimi models and Mimi itself and am wondering if my approach makes sense.  In this case we are going to want to (1) update Mimi (2) tag a new Mimi so that FUND pulls the change when it uses Mimi in Travis (3) update and tag Mimi (4) update the FUND dependency commit in Mimi.  It'll be a really small new tag to Mimi but perhaps that's ok, with an eye towards trying not to let it happen too often :)

Reply:
@davidanthoff @ckingdon95 I fixed this here: https://github.com/mimiframework/Mimi.jl/pull/445.  If you alter the FUND code so that if `save_trials` is false, you don't include the `filename` keyword at all, then we won't need to tag a new version of Mimi yet because that will work with the old version.  Otherwise if you want to set `filename = nothing` I should tag a new version of Mimi.